lib/DDGC/DB/Result/InstantAnswer.pm

```diff
@@ -413,6 +413,10 @@ column updated => {
     set_on_update => 1,
 };
 
+column release_version => {
+    data_type => 'numeric',
+    is_nullable => 1,
+};
```

Comment:
@zachthompson Why not make a relation to the ReleaseVersion table? Just asking out of curiosity - I can definitely imagine it could be useful in the future if we want to see diffs and approve them for release directly on IA Pages, but of course for now it's not necessary!